commons/libs/constants/GenericCodes.ts

```diff
@@ -1,7 +1,8 @@
 // TODO: add  error categories and codes as necessary with further development. remove this comment when the all the codes needed for application are mostly covered
 export const GenericCodes = {
   unauthorized: 401,
-  notFound: 404
+  notFound: 404,
```

Comment:
I feel like these should be upper cased?

Reply:
Yes vai, Converted all to upper case.

You will find with this [Commit](https://github.com/CraftsmenLtd/BloodConnect/pull/113/commits/9931689cd63b25122b4b81b2553290ca9366b22a)